loader/loader.go

```diff
@@ -349,6 +349,10 @@ func (l *Loader) parseGunkPackage(pkg *GunkPackage) {
 		pkg.ProtoName = pkg.Name
 	}
 
+	if len(pkg.Errors) > 0 {
+		return
```

Comment:
shouldn't we report the errors? if they are caught elsewhere, just mention that in a comment here.

Reply:
comment added.